demos/acl-clients.php

```diff
@@ -18,8 +19,7 @@
 // switch on ACL so it will be applied for all models added to persistence from now on
 $app->initAcl();
 
-$app->add([Message::class, 'type' => 'info'])
+Message::addTo($app, [Message::class, 'type' => 'info'])
```

Comment:
```suggestion
Message::addTo($app, ['type' => 'info'])
```

otherwise cl is passed as 1st param!